contrib/linux/actions/rsync.yaml

```diff
@@ -9,21 +9,20 @@
             type: 'string'
             description: 'List of files/directories to to be copied'
             required: true
-            position: 1
+            position: 0
```

Comment:
The positions here should probably just be removed all together.  They get used in the cmd default anyway...seems odd that this even works.  I would think all that data would get passed through twice.